main.go

```diff
@@ -20,6 +20,7 @@ type Resource interface {
 	Conditions() []Condition
 }
 
+//Condition alert conditions
```

Comment:
It's good from a stylistic point of view to have a space after the //
Also there are //go:generate style comments that rely on not having a space.